hydra-configs-torchvision/setup.py

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+from setuptools import find_namespace_packages, setup
+
+setup(
+    name="hydra-configs-torchvision",
+    version="0.7.1",
```

Comment:
to be clear:
`0.7` is the torchvision version, `.1` is the patch version of `hydra-configs-torchvision` (in case we need to release an updated package for torchvision 0.7).

Reasons to update `hydra-configs-torchvision` includes:
- Some change in torchvision 0.7 (unlikely).
- Add additional configs not previously covered to the project
- Some change in configen itself
- Some mistake in the previously released package

Reply:
Correct 👍

Reply:
Cool. We should document out decisions in some md file in the repo.